docs/configuration/animations.md

```diff
@@ -11,11 +11,31 @@ The following animation options are available. The global options for are define
 | `duration` | `number` | `1000` | The number of milliseconds an animation takes.
 | `easing` | `string` | `'easeOutQuart'` | Easing function to use. [more...](#easing)
 | `onProgress` | `function` | `null` | Callback called on each step of an animation. [more...](#animation-callbacks)
-| `onComplete` | `function` | `null` | Callback called at the end of an animation. [more...](#animation-callbacks)
+| `onComplete` | `function` | `null` | Callback called when all animations are completed. [more...](#animation-callbacks)
+| `delay` | `number` | `undefined` | Delay before starting the animations.
+| `loop` | `boolean` | `undefined` | If set to `true`, loop the animations loop endlessly.
```

Comment:
Do you have an intended use case for this? I'm having a hard time envisioning how it'd be used

Reply:
I could imagine a bouncing animation to highlight a point in the graph. Even if we don't have that natively 100%, i can see the value of having loop functionality in the core animation library for 
plugins